src/AllowInheritance.php

```diff
@@ -4,6 +4,7 @@
 
 use Attribute;
 
+/** @psalm-suppress UnusedClass */
```

Comment:
I actually believe this should be annotated as [`@api`](https://psalm.dev/docs/annotating_code/supported_annotations/#api-psalm-api). This communicates the class is part of the public facing API used by installers of this lib.

Reply:
Ah, didnt know this annotation exist.

Regarding failing build: would drop PHP 8.0 then :)